libs/build_dod_dataset.py

```diff
@@ -241,6 +269,7 @@ def get_usa_by_county_df():
     url = '{}/data/cases-jhu/csse_covid_19_daily_reports/{}.csv'.format(
         get_public_data_base_url(), latest.strftime("%m-%d-%Y"))
     raw_df = pd.read_csv(url, dtype={"FIPS": str})
```

Comment:
hmm maybe we need to add the padding?

Reply:
the jhu data does have a few fips (not most of them) without leading zeros i'm pretty sure